stubs/python-jose/jose/backends/__init__.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Callable
+
+from .base import DIRKey as DIRKey
+
+AESKey = Any
+HMACKey = Any
+RSAKey = Any
+ECKey = Any
```

Comment:
Could you add a comment to the code why these are `Any`?

Reply:
I changed the types to better align with what is happening here. Added some comments to clarify. Hope this helps.